include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -39,12 +39,6 @@
   #define ROM_ADDR (CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET)
 #endif
 
-/*
- * MMU currently supports 4 kB translation granule size,
- * so all regions are required to be 4 kB aligned
- */
-#define PAGE_SIZE	0x1000
```

Comment:
I see `PAGE_SIZE` define is removed here, but still used at places in `arm_mmu.c` .. is it defined someplace else as well? just confirming.

Reply:
The rationale here is that we can now use `CONFIG_MMU_PAGE_SIZE` so `PAGE_SIZE` is in theory redundant. The problem is that:
1. `CONFIG_MMU_PAGE_SIZE` can be changed by the user so I added the assert to make sure that `CONFIG_MMU_PAGE_SIZE == PAGE_SIZE` in the MMU code
2. `PAGE_SIZE` and in particular `PAGE_SIZE_SHIFT` is used a lot in the MMU defines so I left that in place.

Reply:
What do you think about that ?
`#define PAGE_SIZE	CONFIG_MMU_PAGE_SIZE`

Reply:
I don't think that's really a solution considering that `PAGE_SIZE_SHIFT` is still a fixed value and that implies a page size of 4K.

Reply:
Seems simplest to be to just use `CONFIG_MMU_PAGE_SIZE`

Reply:
Ok, I replaced `PAGE_SIZE` with `CONFIG_MMU_PAGE_SIZE` leaving the assert in place.

Reply:
We should enforce/check through Kconfig that `CONFIG_MMU_PAGE_SIZE ` is `4kB`.

Reply:
Ok, I'll force it in `aarch64/Kconfig`